configurations/mt-staging.ts

```diff
@@ -115,6 +115,213 @@ const preEngagementConfig: PreEngagementConfig = {
           required: true,
         }
     },
+    {
+      label: 'Select your Age/Età/літа',
+      type: 'SelectItem',
+      attributes: {
+        name: 'age',
+        required: true,
+        readOnly: false,
+      },
+      options: [
+        {
+          value: "",
+          label: ""
+        },
+        {
+          value: "Unborn",
+          label: "Unborn"
+        },
+        {
+          value: "0",
```

Comment:
I'm not sure if you need to add leading zeros here, maybe you can check with a developer like Gian?  in the ChildInfo form these will definitely have leading zeros